src/applications/static-pages/facilities/FacilityDetailWidget.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+import { buildHours } from '../../facility-locator/utils/facilityHours';
+
+export default class FacilityDetailWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    const facilityId = this.props.facility;
+    this.request = apiRequest(
+      `/facilities/va/${facilityId}`,
+      null,
+      this.handleFacilitySuccess,
+      this.handleFacilityError,
+    );
+  }
```

Comment:
The Facility Locator uses a "MockAPI" to make local testing easier/quicker. Would it make sense to use that here as well, instead of the direct `fetch`? https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/facility-locator/api/index.js